cg/meta/demultiplex/status_db_storage_functions.py

```diff
@@ -40,8 +42,12 @@ def store_flow_cell_data_in_status_db(
         LOG.info(f"Flow cell added to status db: {parsed_flow_cell.id}.")
     else:
         LOG.info(f"Flow cell already exists in status db: {parsed_flow_cell.id}.")
+    sample_sheet: File = get_sample_sheets_from_latest_version(
+        flow_cell_id=parsed_flow_cell.id,
+        hk_api=hk_api,
+    )[0]
```

Comment:
I am thinking of what happens if the sample sheet is not in HK (unlikely). This would give a not-so meaningful Housekeeper exception. Should I make a try-except statement around the sample sheet fetching?

Reply:
Yes I think that makes sense here.

Reply:
added